CloudSQLReplicator/google/cloud/pontem/sql/replicator/util/gcp_api_util.py

```diff
@@ -14,55 +14,92 @@
 
 """GCP API utility functions."""
 import logging
-from google.auth import compute_engine
+import google.auth
 from google.cloud import storage
 
+from google.cloud.pontem.sql import replicator
+
+
+
+def _get_user_agent():
+  """Returns user agent based on packagage info."""
+
+  user_agent = 'pontem,{}/{} (gzip)'.format(replicator.__package_name__, replicator.__version__)
```

Comment:
I'm curious. Is there a standard user agent naming convention we follow across tools?